vendor/engines/secure_rooms/spec/services/secure_rooms/access_handlers/occupancy_handler_spec.rb

```diff
@@ -30,15 +30,33 @@
     end
   end
 
+  shared_examples_for "a problem order" do
+    it "sets up a problem order" do
```

Comment:
Maybe `it "results in a problem order"`? This almost makes it feel like you're testing the test setup.